src/wdqms/wdqms.py

```diff
@@ -66,8 +66,18 @@ def __init__(self, inputfiles, wdqms_type, outdir,
         # Drop duplicates
         df_total = df_total.drop_duplicates()
 
+        # Create temporary dataframe of only temperature and q values
+        tq_condition = [self.wdqms_type_dict[self.wdqms_type]['variable_ids']['t'],
+                        self.wdqms_type_dict[self.wdqms_type]['variable_ids']['q']]
+
+        tq_df = df_total.loc[df_total['var_id'].isin(tq_condition)]
+        no_tq_df = df_total.loc[~df_total['var_id'].isin(tq_condition)]
+
         # Adjust relative humidity data
-        df_total = self._genqsat(df_total)
+        tq_df = self._genqsat(tq_df)
+        
+        # Merge the non t and q values back into returned t and q dataframe
+        df_total = pd.concat([new_tq_df, no_tq_df])
```

Comment:
@kevindougherty-noaa 
For line #80,  the dataframe `new_tq_df` is not defined.  Should `new_tq_df` be `tq_df`?

Or, you may want to change line #77 as the following:
```
new_tq_df = self._genqsat(tq_df) 
```